lib/OpenQA/Setup.pm

```diff
@@ -148,6 +149,7 @@ sub read_config ($app) {
             screenshot_cleanup_batch_size => OpenQA::Task::Job::Limit::DEFAULT_SCREENSHOTS_PER_BATCH,
             screenshot_cleanup_batches_per_minion_job => OpenQA::Task::Job::Limit::DEFAULT_BATCHES_PER_MINION_JOB,
             results_min_free_disk_space_percentage => undef,
+            expire_minion_jobs => ONE_DAY * 7,
```

Comment:
```suggestion
            expire_minion_jobs => ONE_WEEK,
```

Reply:
btw, I was also wondering if one week is a good default or if we should even go for a higher default.

Reply:
done